orchestrator/build.gradle

```diff
@@ -18,6 +18,7 @@ plugins {
     id "java"
     id "de.gliderpilot.semantic-release" version "1.4.0"
     id "application"
+    id "nebula.deb" version "8.0.3"
```

Comment:
if you use `nebula.ospackage` instead, you could create an rpm as well as a deb

Reply:
Target for now is Debian but if we can get rpm packages for free at the same time them maybe we should try. That said happy for this to be in a following pr.